docs/design-proposals/cloud-edge-communication/cloud-edge-communication-framework.md

```diff
@@ -0,0 +1,28 @@
+# Arktos Cloud-Edge Communication Framework
+
+## Motivation
+
+In the edge cloud scenario, worker node of arktos can be run on the edge closer to the user, such as CDN machine room of the operator, user data center and other edge sites, and virtual machines, containers and other resources are provided at the edge. Through the edge network with low delay, the operation of delay sensitive application is satisfied, such as interactive live broadcasting, cloud games and real-time trading systems etc..
```

Comment:
This is a very early version of the design proposal.  Per our discussion in the community meeting, we will first merge this initial version and then iterate it.